### PR TITLE
one more os test

### PR DESCRIPTION
**Pull request by oshtaier**
depends on https://github.com/abalakh/airgun/pull/8

### CODE REVIEW

File: tests/foreman/ui_airgun/test_os.py [hunk 1]
```diff
@@ -9,3 +11,16 @@ def test_positive_create(session, name):
...
+
+def test_positive_create_with_ptable(session):
+    name = gen_string('alpha')
+    major_version = gen_string('numeric', 2)
+    ptable = entities.PartitionTable().create()
+    with session:
+        session.os.create_operating_system({
+            'name': name,
+            'major': major_version,
+            'ptable': {'operation': 'Add', 'values': [ptable.name]}
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_os.py]: AFAIK widgetastic provides more "readable" way for tests to use fill:
```python
create_operating_system({
    'name': name,
    'major': major_version,
    'ptable.operation' : 'Add',
    'ptable.values': [ptable.name],
})
```
Although i'm not sure about `ptable.values`, i feel like there's gotta be a way to pass `ptable.name` directly.

Could you please take a look on at least first comment? Second one may appear a separate task to figure out, so it's not a blocker.

**Review by abalakh — COMMENTED** [review]
Pending comment

**Comment by oshtaier** [comment]
>Although i'm not sure about ptable.values, i feel like there's gotta be a way to pass ptable.name directly.

directly where? It is `fill` method for the widget which expects `values` value

**Review by abalakh — DISMISSED** [review]
ACK, that comment is rather idea for investigation

**Comment by oshtaier** [comment]
Added one more case
Please re-review

**Comment by oshtaier** [comment]
```
py.test /home/ashtayer/Desktop/robottelo/tests/foreman/ui_airgun/test_os.py::test_positive_create_with_arch
==================================================================================== test session starts =====================================================================================
platform linux2 -- Python 2.7.14, pytest-3.3.2, py-1.5.2, pluggy-0.6.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/ashtayer/Desktop/robottelo, inifile:
plugins: wait-for-1.0.9, services-1.2.1, mock-1.6.3
collected 1 item                                                                                                                                                                             
2018-02-07 13:08:42 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_os.py .                                                                                                                                                   [100%]

================================================================================= 1 passed in 53.68 seconds ==================================================================================
```

**Review by abalakh — APPROVED** [review]
ACK

**Pull request merged**